app/main.py

```diff
@@ -1,2 +1,61 @@
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+    def __init__(self, capacity: int = 8, load_factor: float = 0.66) -> None:
+        self.capacity = capacity
+        self.load_factor = load_factor
+        self.length = 0
+        self.limit = capacity * load_factor
+        self.hash_table = [[] for _ in range(capacity)]
+
+    def __setitem__(self, key: Hashable, value: Any) -> None:
+        if self.length > self.limit:
+            self.resize()
+        index = self._get_index(key)
+        bucket = self.hash_table[index]
+        for i in range(len(bucket)):
+            if bucket[i][0] == key:
+                bucket[i] = (key, hash(key), value)
+                return
+        bucket.append((key, hash(key), value))
+        self.length += 1
+
+    def __getitem__(self, key: Hashable) -> Any:
+        index = self._get_index(key)
+        for k, h, v in self.hash_table[index]:
+            if k == key:
+                return v
+        raise KeyError(key)
+
+    def __delitem__(self, key: Hashable) -> None:
+        index = self._get_index(key)
+        bucket = self.hash_table[index]
+        for i in range(len(bucket)):
+            if bucket[i][0] == key:
+                del bucket[i]
+                self.length -= 1
+                return
+        raise KeyError(key)
```

Comment:
This method implementation will break the logic of your dictionary. Try to think why.